client/me/purchases/remove-plan-dialog/index.tsx

```diff
@@ -0,0 +1,156 @@
+import { Dialog, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { Fragment } from 'react';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+import { SiteScreenshot } from '../site-screenshot';
+import getPlanFeatures from './get-plan-features';
+
+import './style.scss';
+
+interface RemovePlanDialogProps {
+	closeDialog: () => void;
+	removePlan: () => void;
+	isDialogVisible: boolean;
+	isRemoving: boolean;
+	site: SiteExcerptData;
+}
+
+export const RemovePlanDialog = ( {
+	closeDialog,
+	removePlan,
+	isDialogVisible,
+	site,
+}: RemovePlanDialogProps ) => {
+	const translate = useTranslate();
+
+	/**
+	 * Dialog buttons
+	 */
+	const buttons = [
+		{
+			action: 'cancel',
+			label: translate( 'Keep my plan' ),
+			onClick: closeDialog,
+		},
+		{
+			isPrimary: true,
+			action: 'removePlanAndAllSubscriptions',
+			label: translate( 'Cancel my plan', {
+				comment:
+					'This button removes the active plan and all active Marketplace subscriptions on the site',
```

Comment:
I am not sure what this has to do with marketplace subscriptions or `removePlanAndAllSubscriptions`.  I think it just removes the plan and nothing else.

Reply:
this was updated

Reply:
I'm still seeing the reference to "marketplace subscription" in the above comment, though.

Reply:
Yep, I'm also seeing it. @ciprianimike maybe the commit isn't pushed? 
Either way, I don't think we need this translator comment, since "Cancel my plan" has already been translated in other places since 2 years ago: https://translate.wordpress.com/projects/wpcom/-all-translated/282389/ . So let's remove this comment entirely :)